### ISSUE TITLE
QP: HiGHS returns non optimal solution

### ISSUE THREAD

**Issue by sergey-mogilnikov-kpbs** [ERGO-Code/HiGHS#993]
The problem is follow
```code
NAME Quad3
ROWS
 L  r1
 L  r2
 N  r3
COLUMNS
    x1        r1                  -1   r2                  -1
    x2        r1                  -2   r2                  -1
    x3        r1                  -3   r2                  -1
    x3        r3                   2
    x4        r1                  -4   r2                  -1
    x4        r3                 0.6
    x5        r1                  -5   r2                  -1
    x5        r3                 2.2
    x6        r1                  -6   r2                  -1
    x6        r3                   1
    x7        r1                  -7   r2                  -1
    x7        r3                 4.8
    x8        r1                  -8   r2                  -1
    x8        r3               -2.88
    x9        r1                  -9   r2                  -1
    x9        r3               -0.72
    x10       r1                 -10   r2                  -1
    x10       r3                -9.6
RHS
    RHS1      r1                -100   r2                 -13
QUADOBJ
    x1  x1  2
    x2  x2  2
    x3  x3  2
    x3  x4  0.6
    x3  x5  0.2
    x3  x6  -1
    x4  x4  0.18
    x4  x5  0.06
    x4  x6  -0.3
    x5  x5  2.02
    x5  x6  1.9
    x6  x6  2.5
    x7  x7  10
    x7  x8  -4.8
    x7  x9  -1.2
    x7  x10  -16
    x8  x8  2.88
    x8  x9  0.72
    x8  x10  9.6
    x9  x9  0.18
    x9  x10  2.4
    x10  x10  32
ENDATA
```
HiGHS returns 7.4109585896e-01 instead of -0.31326 (it was checked with cplex and cbc)

**Comment by feldmeier** [comment]
Sorry for the long wait. I'm back to working on HiGHS and the QP solver in particular.
I tried to replicate this issue, but it appears to work fine with the current master/latest branch:

```
Running HiGHS 1.5.1 [date: 2023-04-11, git hash: 95e2ac088]
Copyright (c) 2023 HiGHS under MIT licence terms
QP   issue993 has 2 rows; 10 cols; 20 matrix nonzeros; 22 Hessian nonzeros
0, 10000.000500, 0, 0.001960, 0.000000, 0, 0.000000, 0.000000
17, -0.313254, 3, 0.003062, 0.000000, 0, 0.000000, 1.000000
Model   status      : Optimal
QP ASM    iterations: 17
Objective value     : -3.1326120556e-01
HiGHS run time      :          0.00
```

Are you still getting a wrong result?

**Comment by jajhall** [comment]
There have been a few changes to the QP solver since last summer. Generally they are "cosmetic", but I did fix #958. This was after the creation of #993, but maybe @sergey-mogilnikov-kpbs was using an older version.

**Comment by jajhall** [comment]
It would appear that this issue can be closed

**Comment by jajhall** [comment]
Fair comment. I'll look at this

**Comment by jajhall** [comment]
Correction soon merged into `latest`and will be present in v1.5.2
